sites/public/src/pages/applications/review/terms.tsx

```diff
@@ -94,32 +93,6 @@ const ApplicationTerms = () => {
       })
   }
 
-  const agreeField = [
-    {
-      id: "agree",
-      label: t("application.review.terms.confirmCheckboxText"),
-    },
-  ]
-
-  const content = useMemo(() => {
```

Comment:
Did we confirm with product that it's okay to remove the listing-type-specific text?